index.bs

```diff
@@ -1527,6 +1546,48 @@ example of the latter, when the user is accessing the [=[RP]=] from a given clie
 use a [=roaming authenticator=] which was originally registered with the [=[RP]=] using a different client.
 
 
+### Attestation Preference enumeration (enum <dfn enum>AttestationPreference</dfn>) ### {#attestation}
+
+[=[RPS]=] may use the {{AttestationPreference}} enum to specify their preference regarding attestation types used 
+during credential generation.
+
+<pre class="idl">
+    enum AttestationPreference {
+        "none",
+        "verifiable",
+        "direct"
+    };
+</pre>
+
+<ul>
+    <li><dfn>none</dfn> - indicates that the [=[RP]=] is not interested in attestation, and prefers fast and reliable credential
+      creation over obtaining a verifiable attestation. The client MAY replace the attestation statement generated
+      by the authenticator with a meaningless client-generated attestation statement in this case, for example in order to prevent
+      having to obtain user consent for relaying identifying information to the RP, or to save a roundtrip to a Privacy CA. 
+      The client SHOULD NOT prompt the user for consent to relay identifying information to the RP in this case, and MUST NOT 
+      fail the credential creation operation because of issues with the attestation. For example, if the client normally uses
+      a Privacy CA to anonymize attestations, and the Privacy CA is offline, the client must not fail the credential creation 
+      operation. Instead, it could replace the authenticator-generated attestation with a client-generated attestation.
+
+      This is the default value.
+    <li><dfn>verifiable</dfn> - indicates that the [=[RP]=] prefers an attestation type that leads to verifiable attestation
```

Comment:
Instead of verifiable, can we say Privacy-CA. Attestation produced by authenticators is also verifiable and adds confusion.

Reply:
The "obtain user consent" handling should be identical for "verifiable" and "direct". In both cases the authenticator model will be known to some other party (either the Privacy CA or the RP).
In my opinion Authenticator models satisfying the FIDO requirements (i.e. batch size >= 100000 and others) do not lead to a privacy risk and hence do not require yet another user consent (in addition to the general consent of using the authenticator a creating a credential).